src/helge_helo_klemetsdal_ex/ex04/walker.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Helge Helo Klemetsdal'
+__email__ = 'hegkleme@nmbu.no'
+
+import random as rd
+
+
+class Walker:
+    def __init__(self, initial_position, home_position):
+        self.position = initial_position
+        self.home = home_position
+        self.steps = 0
+
+    def move(self):
+        movement = rd.randint(-1, 1)
+        while movement == 0:
```

Comment:
Not very efficient way, better to get only two random numbers then use if statement or do some math operations